### DISCUSSION TITLE
embla pause

### DISCUSSION THREAD

**Discussion by antoniovassallo** [davidjerleke/embla-carousel#527] [Help]
I am trying writing in english : my Problem that when i am using Embla with some longtext ...the reader doese not have time to read .... what i want do , If the reader on mobile touch the screen , the Silider have to stop or to run again after klicking or touch ? did any one have idea how to acheived that on Embla

**Comment by davidjerleke** [comment]
@antoniovassallo are you using the autoplay plugin?

**Reply by antoniovassallo** [reply]
i add it to the script <script src="https://unpkg.com/embla-carousel-autoplay/embla-carousel-autoplay.umd.js"></script>
and use it like that  
const viewportNode = emblaNode.querySelector('.embla__viewport');
const options = { loop: false };
const plugins = [EmblaCarouselAutoplay()]; // Plugins
        emblaApi = EmblaCarousel(viewportNode, options, plugins);

**Reply by antoniovassallo** [reply]
i forget to say that the embla interface used from Mobirise Software example  https://www.villa-kritikos.de/assets/embla/script.js
and https://www.villa-kritikos.de/assets/embla/embla.min.js

**Reply by MarlonPassos-git** [reply]
@antoniovassallo  Have you tried using the stopOnInteraction option? Well, maybe it will solve your problem.

```js
const viewportNode = emblaNode. querySelector('.embla__viewport');
const opções = { loop: false };
plugins const = [EmblaCarouselAutoplay({ stopOnInteraction: true })]; // Plugins
emblaApi = EmblaCarousel(viewportNode, opções, plugins);
```
